src/main/scala/com/scality/clueso/query/MetadataQueryExecutor.scala

```diff
@@ -0,0 +1,255 @@
+package com.scality.clueso.query
+
+import java.util.concurrent.Executors
+import java.util.concurrent.atomic.AtomicBoolean
+import java.util.{concurrent => juc}
```

Comment:
Renames the name of the package to juc, in this case anything that has juc.X means java.util.concurrent.X. This is a nice way to tidy conflicts in namespaces where same class names are used and also to highlight in the code the Java concurrency utils are being used instead of Scala. 

Same as Python's 'import numpy as np'